Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM golang:1.13.1-alpine3.10 as builder
+
+RUN apk add --update --no-cache ca-certificates tzdata git make && update-ca-certificates
+
+ADD . /opt
+WORKDIR /opt
+
+RUN git update-index --refresh; make observatorium
+
+FROM scratch as runner
```

Comment:
I don't really like running scratch, it's super hard to debug inside the container if something is wrong. Personally I'd prefer alpine, but I can also live with busybox.